frontend/src/metabase/ui/components/buttons/ActionIcon/ActionIcon.styled.tsx

```diff
@@ -4,12 +4,13 @@ export const getActionIconOverrides =
   (): MantineThemeOverride["components"] => ({
     ActionIcon: {
       variants: {
-        subtle: theme => ({
+        // Default variant is "subtle"
+        subtle: () => ({
           root: {
-            color: theme.fn.themeColor("text-light"),
+            color: "var(--mb-color-text-tertiary)",
```

Comment:
Style the dashcard action menu icon 
![image](https://github.com/user-attachments/assets/ef091961-e313-4a6a-844e-40aa9c94b45e)